scripts/create_cluster_entity.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+SCRIPTS_DIR="$(cd $(dirname "${BASH_SOURCE}") && pwd)"
+
+source ${SCRIPTS_DIR}/common.sh
+source ${SCRIPTS_DIR}/play-demo.sh
+
+# Parameters to play the script
+TYPE_SPEED=200
+NO_WAIT=true
+
+# Script parameters
+PRIMAZA_URL=${PRIMAZA_URL:-localhost:8080}
+p "Primaza server: ${PRIMAZA_URL}"
+CFG=$(kind get kubeconfig)
+KIND_URL=$(kubectl config view -o json | jq -r '.clusters[0].cluster.server')
```

Comment:
if we have more than one cluster it could not take the good one. Can we take the one the current-context is pointing to?

Reply:
yes, exactly :-)

Reply:
Fixed with: 1891b9b